src/Bundler.js

```diff
@@ -87,16 +87,19 @@ class Bundler extends EventEmitter {
     this.packagers.add(type, packager);
   }
 
-  loadPlugins() {
+  loadPlugins(location = this.mainFile) {
     try {
-      let pkg = localRequire('./package.json', this.mainFile);
+      let pkg = localRequire('./package.json', location);
       let deps = Object.assign({}, pkg.dependencies, pkg.devDependencies);
       for (let dep in deps) {
         if (dep.startsWith('parcel-plugin-')) {
-          localRequire(dep, this.mainFile)(this);
+          localRequire(dep, location)(this);
         }
       }
     } catch (err) {
+      if (err.code === 'MODULE_NOT_FOUND' && location.length > 1) {
```

Comment:
we probably only want to do this if the require of `package.json` failed, not one of plugins.

what if we used the `config.load` util function that already exists to load the package.json file instead? that will already do the traversing for us.

Reply:
Ow didn't know that i'll change it to config.load